pkg/resources/external_function.go

```diff
@@ -172,7 +169,6 @@ var externalFunctionSchema = map[string]*schema.Schema{
 		Type:        schema.TypeString,
 		Optional:    true,
 		Default:     "user-defined function",
-		ForceNew:    true,
```

Comment:
It also changes behavior. Please at least add this to the migration guide to make users aware of the change?

Reply:
added to migration guide